roles/lib_openshift/src/doc/label

```diff
@@ -0,0 +1,64 @@
+# flake8: noqa
+# pylint: skip-file
+
+DOCUMENTATION = '''
+---
+module: oc_label
+short_description: Create, modify, and idempotently manage openshift labels.
+description:
+  - Modify openshift labels programmatically.
+options:
+  state:
+    description:
+    - State represents whether to create, modify, delete, or list
+    default: present
+    choices: ["present", "absent", "list", "add"]
+    aliases: []
+  kubeconfig:
+    description:
+    - The path for the kubeconfig file to use for authentication
+    required: false
+    default: /etc/origin/master/admin.kubeconfig
+    aliases: []
+  debug:
+    description:
+    - Turn on debug output.
+    required: false
+    default: False
+    aliases: []
+  kind:
```

Comment:
The `AnsibleModule` spec for `kind` defines it as such:

```python
    kind=dict(
        default=None,
        type='str',
        required=True,
        choices=['node', 'pod', 'namespace']
),
```

However the `DOCUMENTATION` does not match

Param |`AnsibleModule` spec | `DOCUMENTATION`
------------ | ------------ | -------------
default | `None` | **node**
required | `True` | ⁿ/ₐ